content/blog/2020-announcing-epam-as-gold-sponsor.md

```diff
@@ -0,0 +1,20 @@
+Title: Announcing EPAM as a Gold Sponsor
+Date: 2020-09-19 17:00
+Author: Nikhil Maan (@Sc0rpi0n101)
+Category: 2020
+Slug: sponsor-announcement-epam
+Summary: We are happy to introduce **EPAM** as a Gold Sponsor for PyCon India 2020. 
+
+We are happy to introduce **EPAM** as a Gold Sponsor for PyCon India 2020.
+
+<br>
+<div class="text-center">
+  <a href="https://welcome.epam.in/" target="_blank">
+    <img src="https://user-images.githubusercontent.com/3849885/93663821-126f4600-fa88-11ea-982a-2d3feaba4843.png" alt="EPAM">
```

Comment:
This change. Please use https://in.pycon.org/2020/assets/images/sponsors/epam.png

Reply:
Done